app/controllers/payform_sets_controller.rb

```diff
@@ -12,14 +12,19 @@ def show
     @payform_set = PayformSet.find(params[:id])
     payform_items = @payform_set.payforms.map(&:payform_items).flatten
     @grouped_items = payform_items.group_by{|pi| pi.category.name}
-
+    @sorted_payforms = @payform_set.payforms.delete_if{|payform| payform.hours == 0}.sort_by{|payform| payform.user.last_name}.sort_by{|payform| payform.date}
```

Comment:
Yea I know, I figured we'd do a `rubocop` purge eventually so didn't want to be picky.